lib/transport_web/api/views/dataset_view.ex

```diff
@@ -6,6 +6,9 @@ defmodule TransportWeb.API.DatasetView do
   end
 
   def render(_conn, %{errors: errors}) do
-    Poison.encode(errors)
+    case Poison.encode(errors) do
```

Comment:
It changes quite a bit the return of this function, was it a bug before?

Reply:
It is weird as it was not failing, but it should have! I say it was a bug before.